src/access/AccessControl.cpp

```diff
@@ -23,92 +23,153 @@ namespace {
 using chip::FabricIndex;
 using namespace chip::Access;
 
-// Avoid GetAccessControl returning nullptr before SetAccessControl is called.
-class UnimplementedDataProvider : public AccessControlDataProvider
+AccessControl defaultAccessControl;
+AccessControl * globalAccessControl = &defaultAccessControl;
+
+static_assert((unsigned(Privilege::kAdminister) & unsigned(Privilege::kManage)) == 0, "Privilege bits must be unique");
```

Comment:
I am wondering if `&` all bits in a single `static_assert` does the same thing?

Reply:
We can put them all in one static_assert, but may I ask why?

Reply:
I put them all into one, on the theory that it's harder to accidentally drop one of the checks, even though code review should question such a weird change if it ever happens.